deploy-continuous-integration.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -xe
+
+# Deploys the Continuous Integration service CloudFormation stack.
```

Comment:
If you're feeling extra autonomous, you can make _another_ cloudformation stack that deploys a CodePipeline pipeline which is responsible for updating the `javabuilder-continuous-integration` stack. If you do this, then you can edit the "continuous-integration.yml" file, merge to main, and have it automatically update the CI pipeline. 

The new "CI-Deployer" stack & pipeline is incredibly unlikely to ever need updating, whereas the javabuilder-continuous-integration stack will see changes from time to time.

The advantage is that we eliminate a scenario where a developer needs powerful 'deployer' level permissions.

One gotcha is that you want to avoid merging CI and App updates in the same merge, because you'll have a race condition where the pipeline is being updated at the same time the app is being deployed. I have not looked into ways to avoid this or make it safer. It is possible to have your App pipeline run where early steps are defined by the previous commit, and later steps are defined by the new commit. I've always done more frequent, smaller deploys to production so it was rarely an issue.